### PR TITLE
fine tune readme

### PR DESCRIPTION
**Pull request by rootfs**
@SeineSailor what do you think about this?

### CODE REVIEW

**Comment by rootfs** [comment]
@SeineSailor review this

**Comment by rootfs** [comment]
@SeineSailor Please review this pull request and provide a summary of the changes.

File: README.md [hunk 1]
```diff
@@ -6,12 +6,14 @@
 ## Overview
 
 SeineSailor is an AI-powered copilot for project managers, designed to assist with code reviews, project discussions,
-and knowledge transfer. It leverages the power of WatsonX's `llama3` and `mixtral-8x7b` models to provide intelligent
+and knowledge transfer. It leverages the power of `llama3` and `mixtral-8x7b` models, hosted by Cloud providers such as IBM WatsonX, to provide intelligent
 and context-aware support throughout the development lifecycle.
 
+SeineSailor is inspired by many research works [[1](https://www.computer.org/csdl/proceedings-article/issre/2023/159400a647/1RKjp8pPMHu), [2](https://arxiv.org/abs/2312.15698)] and open source projects[[3](https://github.com/anc95/ChatGPT-CodeReview), [4](https://github.com/coderabbitai/ai-pr-reviewer)]
```
Comment (wenboown) [inline README.md:12]: good addition!

Comment (wenboown) [reply README.md:12]: @SeineSailor what do you think about this?

Comment (wenboown) [reply README.md:12]: @SeineSailor try again

**Comment by rootfs** [comment]
@SeineSailor what do you think about this?

**Comment by rootfs** [comment]
@SeineSailor what do you think about this?

**Pull request merged**